File: Fuente/Implementación/api-autenticacion/Autenticacion.Dominio/Modelo/Command/LoginResponse.cs
using Autenticacion.Dominio.Modelo.Queries;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace Autenticacion.Dominio.Modelo.Command
{
    public class LoginResponse
    {
        [Required]
        public bool Autenticado { get; set; }

        public TokenJWT TokenJWT { get; set; }

        [Required]
        public string Mensaje { get; set; }

        [Required]
        public bool Bloqueado { get; set; }

        public string URLdesbloqueo { get; set; }

        public UsuarioQuery DatosUsuario { get; set; }


    }

}

File: Fuente/Implementación/portalWeb/src/app/shared/model/account/query/login-response.ts
import { TokenJWT } from "../token-jwt";
import { UserQuery } from "./user-query";

export class LoginResponse {

    public autenticado : boolean;
    public tokenJWT : TokenJWT;
    public idAplicacion : string;
    public token : string;
    public tokenValido : boolean;
    public mensaje : string;
    public bloqueado : boolean;
    public urLdesbloqueo : string
    
    public datosUsuario : UserQuery;
}

